tools/kimchi-visu/Cargo.toml

```diff
@@ -13,8 +13,8 @@ license = "Apache-2.0"
 path = "src/lib.rs"
 
 [dependencies]
-ark-ec = "0.3.0"
-ark-ff = "0.3.0"
+ark-ec = "0.4.2"
```

Comment:
Note: would be good to backport also the workspace usage as we do have in master.

Reply:
Yes! The way it is now in compatible is very annoying. But I don't want to have more conflicts as of now, so can do it later.